sdk/monitor/azure-monitor-opentelemetry-exporter/tests/logs/test_logs.py

```diff
@@ -173,13 +196,26 @@ def test_span_to_envelope_partA_default(self):
         self.assertEqual(envelope.tags.get("ai.internal.nodeName"), envelope.tags.get("ai.cloud.roleInstance"))
         self._log_data.log_record.resource = old_resource
 
-    def test_log_to_envelope_properties(self):
+    def test_log_to_envelope_log(self):
         exporter = self._exporter
         envelope = exporter._log_to_envelope(self._log_data)
         record = self._log_data.log_record
+        self.assertEqual(envelope.data.base_type, 'MessageData')
         self.assertEqual(envelope.data.base_data.message, record.body)
-        self.assertEqual(envelope.data.base_data.severity_level, _get_severity_level(record.severity_number))
+        self.assertEqual(envelope.data.base_data.severity_level, 2)
+        self.assertEqual(envelope.data.base_data.properties["test"], "attribute")
+
+    def test_log_to_envelope_exception(self):
+        exporter = self._exporter
+        envelope = exporter._log_to_envelope(self._exc_data)
+        self.assertEqual(envelope.data.base_type, 'TelemetryExceptionData')
+        self.assertEqual(envelope.data.base_data.severity_level, 4)
         self.assertEqual(envelope.data.base_data.properties["test"], "attribute")
+        self.assertEqual(len(envelope.data.base_data.exceptions), 1)
+        self.assertEqual(envelope.data.base_data.exceptions[0].type_name, "ZeroDivisionError")
+        self.assertEqual(envelope.data.base_data.exceptions[0].message, "division by zero")
+        self.assertTrue(envelope.data.base_data.exceptions[0].has_full_stack)
```

Comment:
why are we using `asserEqual` as opposed to the pytest syntax to use `assert`

Reply:
Created a tracking issue in case we want to tackle this in the future https://github.com/Azure/azure-sdk-for-python/issues/23648